awx_collection/plugins/modules/tower_workflow_template.py

```diff
@@ -143,6 +145,12 @@ def main():
         supports_check_mode=False
     )
 
+    module.deprecate(msg=(
+        "This module is replaced by the combination of tower_workflow_job_template and "
+        "tower_workflow_job_template_node. This uses the old tower-cli and wll be "
+        "removed in 2022."
+    ), version='4.2.0')
```

Comment:
@gundalow this was discussed in the `#ansible-devel` IRC channel.

This is the most compliant that I can make it, adhering to the old 4-release policy.

We plan to have 2 version streams, one for upstream and one for downstream, in both cases corresponding to server version, and we will only reference downstream versions. Anyway, I can't predict version numbers 2 years from now with certainty, so it's a wild guess so that I can pass sanity tests.

@wenottingham might also want to know what I'm trying to do. This is what I think the final form for all our deprecations should be when we inaugurate the downstream.

Reply:
If we have our own separate collection, why do we need to follow the prior ansible deprecation policy?

Reply:
I would just say "in a future release"

Reply:
> If we have our own separate collection, why do we need to follow the prior ansible deprecation policy?

Our content is going in the ACD. When people upgrade Ansible 2.9 to 2.10, _this_ content is what they'll be running if their playbooks use the modules here.

And I can't say "in a future release". Tests will fail if I don't give it a version. These tests are explicitly designed to enforce standards among collections. I'm not interested in fighting that battle, and frankly, I'm not eager to the action this statement is warning about.